frontend/src/pages/Accounts/components/AccountDetail.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import styled, { css } from 'react-emotion';
+
+import { getAccount } from '../../../data/accounts';
+
+const BodyContainer = styled('div')({
+  maxWidth: 880,
+  margin: '0 auto',
+  padding: '0 40px',
+});
+
+const Spacer = styled('div')({
+  marginTop: 40,
+  color: '#666',
+  '& p': {
+    marginBottom: 20,
+    fontSize: 20,
+  },
+});
```

Comment:
I think your `Spacer` component can just be `const Spacer = styled('div')({ height: 40 });`